app/templates/partials/forms/select.html

```diff
@@ -1,4 +1,4 @@
-<select class='input input--select {{ exclusive_class }}' id='{{select.name}}' name='{{select.name}}'>
+<select class='input input--select {{ select.classes }} {{ exclusive_class }}' id='{{select.name}}' name='{{select.name}}'>
```

Comment:
Can't tell which is better but just putting it out there, feel free to ignore.
``` Jinja
<select class='input input--select {{ select.classes ~' '~ exclusive_class }}' id='{{select.name}}' name='{{select.name}}'>
```

Reply:
What do the tildes do? I can't find anything about it in the nunjucks docs

Reply:
I think the original reads better

Reply:
@bameyrick The tilde is documented at http://jinja.pocoo.org/docs/2.10/templates/#other-operators

Reply:
Ah thank you, I thought it might have done something with whitespace management, but as it only concatenates strings I agree the original is more legible